build.sh

```diff
@@ -1004,6 +1004,8 @@ EOM
           mkdir -p include/util/apple
           cp Realm/*.hpp include
           cp Realm/ObjectStore/src/*.hpp include
+          cp Realm/ObjectStore/src/sync/*.hpp include/sync
```

Comment:
These two lines will fail because `include/sync` doesn't exist. The `mkdir -p` lines above can be combined into this, which will fix the issue: `mkdir -p include/impl/apple include/util/apple include/sync/impl`